api/millenium_patron.py

```diff
@@ -171,7 +171,11 @@ def family_name_match(self, actual_name, supposed_family_name):
         """Does `supposed_family_name` match `actual_name`?"""
         if actual_name is None or supposed_family_name is None:
             return False
-        actual_family_name = actual_name.split(',')[0]
+        if actual_name.find(',') != -1:
+            actual_family_name = actual_name.split(',')[0]
+        else:
+            actual_name_split = actual_name.split(' ')
+            actual_family_name = actual_name_split[len(actual_name_split)-1]
```

Comment:
This makes sense; you should use the shorthand `actual_name_split[-1]` here and add a test for a name like "Anthony Michael Hall".